config/routes.rb

```diff
@@ -46,7 +46,7 @@
       get 'caseload/global' => 'caseload_global#index'
     end
 
-    resources :prisoners, only: [:show] do
+    resources :prisoners, only: [:show, :review_case_details] do
```

Comment:
That doesn't do anything, does it? `only` just affects the built-in resourcey actions  index/create/update/new/edit/delete, custom stuff doesn't go in here. You're doing it the right way below on line 65.